packages/blueprints-integration/src/rundown.ts

```diff
@@ -283,6 +284,18 @@ export interface PieceTransition {
 	duration: number
 }
 
+export interface IBlueprintDirectPlayAdLibPiece {
```

Comment:
A small note, to better describe that `IBlueprintDirectPlayAdLibPiece` and `IBlueprintDirectPlayAdLibAction` are related, I usually do something like this:

```
interface myInterfaceBase {
  type: myInterfaceType
}
enum myInterfaceType {
  A: 'a',
  B: 'b'
}
interface myInterfaceA extends myInterfaceBase {
  type: myInterfaceType.A
  ...otherProperties
}
interface myInterfaceB extends myInterfaceBase {
  type: myInterfaceType.B
  ...otherProperties
}
type myInterface = myInterfaceA | myInterfaceB

```

I think that both extending a base interface and using an enum for the `type` helps with understanding, using and extending the interfaces.